chall01/llahti.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+import sys
+
+morse = [".-",
+"-...",
+"-.-.",
+"-..",
+".",
+"..-.",
+"--.",
+"....",
+"..", 
+".---",
+"-.-",
+".-..",
+"--",
+"-.",
+"---",
+".--.",
+"--.-",
+".-.",
+"...",
+"-",
+"..-",
+"...-",
+".--",
+"-..-",
+"-.--",
+"--.."]
+
+if len(sys.argv) != 2 or sys.argv[1] == "":
+    print("usage: ./xlogin.py <a-zA-Z string>")
+    exit()
+
+string = ""
+
+for c in sys.argv[1]:
+    if c.isalpha() == False and c != ' ':
+        print("usage: ./xlogin.py <a-zA-Z string>")
+        exit()
+    if c == ' ':
+        string += ' '
+    elif c.islower():
+        string += morse[ord(c) - ord('a')]
+    else:
+        string += morse[ord(c) - ord('A')]
+
```

Comment:
A good practice is to protect your python scripts that are meant to be run from running on import.
It's done by
```python
if __name__ == "__main__":
```
and then your code (or even better would be to put your code to function called main() and call it here)